udata/search/result.py

```diff
@@ -48,14 +48,14 @@ def get_ids(self):
             return []
 
     def get_objects(self):
-        if not self._objects:
+        if not self.mongo_objects:
```

Comment:
Called when mongo_objects is None because no result has been found, resulting in `get_ids` getting called and crashing.